config/configmap/inferenceservice.yaml

```diff
@@ -61,24 +61,13 @@ data:
           }
         },
         "pytorch": {
-          "v1" : {
-            "image": "gcr.io/kfserving/pytorchserver",
-            "defaultImageVersion": "latest",
-            "defaultGpuImageVersion": "latest-gpu",
-            "supportedFrameworks": [
-              "pytorch"
-            ],
-            "multiModelServer": false
-          },
-          "v2" : {
-            "image": "pytorch/torchserve-kfs",
-            "defaultImageVersion": "0.4.1",
-            "defaultGpuImageVersion": "0.4.1-gpu",
-            "supportedFrameworks": [
-              "pytorch"
-            ],
-            "multiModelServer": false
-          }
+          "image": "jagadeeshj/torchserve-kfsv2",
```

Comment:
This will need to changed to published image path, please add a TODO to update once the image is ready